cmake/modules/SearchRootCoreDeps.cmake

```diff
@@ -51,21 +52,23 @@ if(CMAKE_VERSION VERSION_GREATER_EQUAL 3.14)
       set(PYTHON_EXECUTABLE "${Python2_EXECUTABLE}" CACHE INTERNAL "" FORCE)
       set(PYTHON_INCLUDE_DIRS "${Python2_INCLUDE_DIRS}" CACHE INTERNAL "" FORCE)
       set(PYTHON_LIBRARIES "${Python2_LIBRARIES}" CACHE INTERNAL "" FORCE)
+      set(PYTHON_VERSION_STRING "${Python2_VERSION}" CACHE INTERNAL "" FORCE)
       set(PYTHON_VERSION_MAJOR "${Python2_VERSION_MAJOR}" CACHE INTERNAL "" FORCE)
       set(PYTHON_VERSION_MINOR "${Python2_VERSION_MINOR}" CACHE INTERNAL "" FORCE)
-      set(PYTHON_VERSION_STRING "${Python2_VERSION_MAJOR}_${Python2_VERSION_MINOR}" CACHE INTERNAL "" FORCE)
+      set(PYTHON_UNDER_VERSION_STRING "${Python2_VERSION_MAJOR}_${Python2_VERSION_MINOR}")
```

Comment:
Why not here to cache it?

Reply:
because we need it only internally when I run the loops in all the PyROOT CMakeLists.txt

Reply:
I was worried if it is visible out side of cmake/modules/SearchRootCoreDeps.cmake.

Reply:
when I built everything was fine. Do you want me to put it back anyway?

Reply:
but then why in line 43 and 102 it is done in different way? It is confusing...

Reply:
because I forgot to remove it :) just changed